.github/workflows/golangci-lint.yml

```diff
@@ -23,5 +23,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6.5.2
```

Comment:
We can update the version as well

Reply:
@ofekshenawa I agree with @elena-kolevska. We can simply update the version in this PR. My initial idea was to update the config in #3328 , but since this is already done here, let's just update the version.